src/components/exchange/ExchangeAssetList.tsx

```diff
@@ -251,19 +263,14 @@ const ExchangeAssetList = (
         ...item,
         data: data.map(rowData => ({
           ...rowData,
-          contextMenuProps: contextMenuProps(
-            rowData ?? store.getState().data.genericAssets?.[rowData.uniqueId],
```

Comment:
🤔 looks like if `item.data` exists it's `EnrichedExchangeAsset[]`, so if the array has values and the `.map` executes, `rowData.uniqueId` should exist, right?

If `rowData` was for some reason undefined or null, it would throw with "Cannot read property undefined of undefined/null" or whatever.

So I _think_ for extra safety you'd want to do `rowData?.uniqueId`, or filter out falsy values from the array before mapping over them.